packages/common/src/compiler/translator.mock.ts

```diff
@@ -54,6 +54,26 @@ export const DBT_METRIC: DbtMetric = {
     refs: [['myTable']],
 };
 
+export const DBT_METRIC_WITH_SIMILAR_NAME = {
```

Comment:
where is this being used ?

Reply:
Oh it's not being used by the test!! I thought it was needed to render the sql for the derived metric `metric_11 / metric_1` but I guess it isn't used in the test. I'll remove it!